drivers/display/display_stm32_ltdc.c

```diff
@@ -57,13 +57,13 @@ LOG_MODULE_REGISTER(display_stm32_ltdc, CONFIG_DISPLAY_LOG_LEVEL);
 #if defined(CONFIG_HAS_CMSIS_CORE_M)
 #include <cmsis_core.h>
 
-#if __DCACHE_PRESENT == 1
+#if defined(CONFIG_DCACHE)
```

Comment:
The `CONFIG_DCACHE` is always defined, regardless if it is on or off.

Reply:
`#if defined(CONFIG_DCACHE)` is a stardand usage for boolean type kconfig.

Reply:
If `CONFIG_DCACHE` set to `y`, it will not be add to `build/zephyr/include/generated/autoconf.h` as below:
```
 #define CONFIG_DCACHE 1
 #define CONFIG_ICACHE 1
```
If `CONFIG_DCACHE` set to `n`, there no `CONFIG_DCACHE` definition in `build/zephyr/include/generated/autoconf.h`.

Disable dcache is the first solution to solve the screen flicker issue. Now we have second and more better solution, `disable FMC bank1`. So, `CONFIG_DCACHE` is not required in here, I can just remove them.

Reply:
> If CONFIG_DCACHE set to n, there no CONFIG_DCACHE definition in build/zephyr/include/generated/autoconf.h.

Indeed, sorry, my bad.

> Disable dcache is the first solution to solve the screen flicker issue. 

Might not be an option for some STM32 MCUs.